unlock-app/src/components/interface/GlobalWrapper.tsx

```diff
@@ -22,17 +22,15 @@ export const GlobalWrapper = ({ children }: GlobalWrapperProps) => {
   }, [])
 
   return (
-    <>
-      {children && (
-        <ConfigContext.Provider value={config}>
-          <WedlockServiceContext.Provider value={wedlockService}>
-            <ProviderContext.Provider value={{ provider, setProvider }}>
-              <Authenticate>{children}</Authenticate>
-            </ProviderContext.Provider>
-          </WedlockServiceContext.Provider>
-        </ConfigContext.Provider>
-      )}
-    </>
+    <UnlockUIProvider Link={NextLink}>
```

Comment:
is not required anymore to check for `children`?

Reply:
Nope. I don't know why that was the case as it was old code.

Reply:
Nice!